tests/fixtures_jlap.py

```diff
@@ -53,6 +53,17 @@ def download_file(subdir, name):
     return flask.send_from_directory(Path(base, subdir), name)
 
 
+@app.route("/none-accept-ranges/")
+def none_accept_ranges():
+    """
+    Returns an empty request with "Accept-Ranges" set to "none"
+    """
+    response = flask.Response("test content test content test content")
+    response.headers["Accept-Ranges"] = "none"
+
+    return response
```

Comment:
This may work. We will have to check that flask ignores the range request headers, and returns all the bytes even if we ask for only some of the bytes.
The test suite also contains a `http_test_server` fixture based on Python's `simplehttpserver`. It wouldn't return `accept-ranges: none`, which would be OK since we ignore that response header, but it may ignore the range request headers.